.travis.yml

```diff
@@ -29,7 +29,11 @@ install:
   - pip install --user -r requirements.txt
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then export PATH=$PATH:$HOME/Library/Python/2.7/bin; export DEVITO_OPENMP=0; export DEVITO_ARCH=clang; fi
 
+before_script:
+  - echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
+
 script:
+  - ./docs/deploy.sh
```

Comment:
Just a minor point, but could we please move this at the end again? It's slightly annoying having to scroll past the verbose output of the deploy script when checking for bugs.